litestar/contrib/minijinja.py

```diff
@@ -174,6 +174,31 @@ def register_template_callable(
         """
         self.engine.add_global(key, pass_state(template_callable))
 
+    def render_string(self, template_string: str, context: Mapping[str, Any]) -> str:
+        """Render a template from a string with the given context.
+
+        Args:
+            template_string: The template string to render.
+            context: A dictionary of variables to pass to the template.
+
+        Returns:
+            The rendered template as a string.
+
+        Raises:
+            TemplateNotFoundException: if no template is found.
+        """
+
+        if template_string is None:
+            raise TemplateNotFoundException("No template string provided", template_name="template_from_string")
```

Comment:
Is this needed? The check for whether `template_string` would have happened before this is ever called.